Fix broken link to met-data documentation

dlebauer opened the pull request:
- fix broken link
- Updated pecan_standards section to standardize heading level, e.g. 'input standards' and 'output standards' are now at the same level, and will show up in TOC on left. Previously the entire page had the title "Time Standards" which was confusing

dlebauer committed:
Fix broken link to met-data documentation

infotroph commented on modules/data.atmosphere/README.md:
```diff
@@ -32,7 +32,7 @@ The PEcAn.data.atmosphere package is 'standalone'.
 ## PEcAn variable names
 
-See https://pecanproject.github.io/pecan-documentation/latest/met-data.html
+For the most updated list, see https://pecanproject.github.io/pecan-documentation/develop/time-standard.html#input-standards
```
I think links in most other docs point to the main branch (confusingly `latest` on github.io and `main` on github.com). Is it important for this one to be the develop version?

dlebauer replied:
Ah, I assumed latest == develop. Happy to change, but it seems that main can be years out of date and in practice it seems most people use develop. Probably more trouble than it is worth to parameterize this.

dlebauer replied:
fixed, I'm not 100% if I understand correctly, but only `master` and `develop` and other branch names in the URL point to an existing page in the documentation.

dlebauer committed:
Consistent indents in 02_pecan_standards.Rmd

Updated so that indents are consistent, e.g. 'input standards' and 'output standards' are at the same level, and will show up in TOC on left

dlebauer commented on modules/data.atmosphere/README.md:
```diff
@@ -32,7 +32,7 @@ The PEcAn.data.atmosphere package is 'standalone'.
 ## PEcAn variable names
 
-See https://pecanproject.github.io/pecan-documentation/latest/met-data.html
+For the most updated list, see https://pecanproject.github.io/pecan-documentation/develop/time-standard.html#input-standards
```
Fixed: 

```suggestion
For the most updated list, see https://pecanproject.github.io/pecan-documentation/master/time-standard.html#input-standards
```

dlebauer committed:
Update modules/data.atmosphere/README.md

dlebauer commented on modules/data.atmosphere/README.md:
```diff
@@ -32,7 +32,7 @@ The PEcAn.data.atmosphere package is 'standalone'.
 ## PEcAn variable names
 
-See https://pecanproject.github.io/pecan-documentation/latest/met-data.html
+For the most updated list, see https://pecanproject.github.io/pecan-documentation/master/time-standard.html#input-standards
```
```suggestion
For the most updated list, see https://pecanproject.github.io/pecan-documentation/latest/time-standard.html#input-standards
```

dlebauer committed:
Update modules/data.atmosphere/README.md

dlebauer commented on modules/data.atmosphere/README.md:
```diff
@@ -32,7 +32,7 @@ The PEcAn.data.atmosphere package is 'standalone'.
 ## PEcAn variable names
 
-See https://pecanproject.github.io/pecan-documentation/latest/met-data.html
+For the most updated list, see https://pecanproject.github.io/pecan-documentation/latest/time-standard.html#input-standards
```
```suggestion
For the most updated list, see https://pecanproject.github.io/pecan-documentation/master/time-standard.html#input-standards
```

dlebauer committed:
Update modules/data.atmosphere/README.md

infotroph commented on book_source/03_topical_pages/02_pecan_standards.Rmd:
```diff
@@ -10,11 +10,11 @@ Internal PEcAn standard time follows ISO_8601 format for dates and time (https:/
+## Input Standards
 
-#### Meterology Standards 
+### Meterology Standards 
```
```suggestion
### Meteorology Standards 
```

infotroph committed:
Update book_source/03_topical_pages/02_pecan_standards.Rmd

infotroph committed:
Update modules/data.atmosphere/README.md

Co-authored-by: David LeBauer <dlebauer@gmail.com>
